dive_core/perf_metrics_data.cpp

```diff
@@ -490,6 +471,17 @@ std::unique_ptr<PerfMetricsData> data)
     result->Update(std::move(data));
     return result;
 }
+
+std::unique_ptr<PerfMetricsDataProvider> PerfMetricsDataProvider::CreateForTest(
+std::unique_ptr<PerfMetricsData>  data,
+std::unique_ptr<AvailableMetrics> desc)
+{
+    auto result = std::unique_ptr<PerfMetricsDataProvider>(new PerfMetricsDataProvider);
```

Comment:
could call `auto result = Create(data);` just in case we may add code in `PerfMetricsDataProvider::Create` in the future.

Reply:
Updated.